.github/workflows/test-and-lint-docker-build.yml

```diff
@@ -0,0 +1,136 @@
+---
+name: Runs linting and tests
+on:
+  workflow_call:
+
+jobs:
+  linting:
```

Comment:
**question (non-blocking)**: I don't know enough about the granularity available in Github actions jobs, but it looks to me like there is no docker differences in the `linting` job which happens inside the Github runner rather than in our docker image. Is it worth splitting that job out so that it doesn't have to be repeated between these two invocations?

Reply:
While they are added in two places, they only execute once regardless. The conditional logic [here](https://github.com/GSA-TTS/FAC/blob/64ea9e3ef2858d53add14ac116db025dbfafb75d/.github/workflows/triggers-PR.yml#L47) and [here](https://github.com/GSA-TTS/FAC/blob/64ea9e3ef2858d53add14ac116db025dbfafb75d/.github/workflows/triggers-PR.yml#L41) determine which `test.yml` should be run and lint/test accordingly.

```mermaid
graph TD;
  A[Any of the 4 files changes]-->B[Lint]
  A-->C[Build container and run django/a11y]
```
```mermaid
graph TD;
  A[No changes to dependencies files]-->B[Lint]
  A-->C[Pull container from gchr and run django/a11y]
```